Vagrantfile.example

```diff
@@ -16,17 +16,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
      config.hostmanager.manage_host = true
  end
 
- # Comment this out to prevent resizing the root filesystem to fill the base box disk
- config.vm.provision "shell", inline: "sudo xfs_growfs /dev/vda3"
- # the command args above must match the filesystems and devices used in the base box
-
- # Comment this line if you would like to disable the automatic update during provisioning
- config.vm.provision "shell", inline: "sudo dnf upgrade -y"
-
- config.vm.provision "ansible" do |ansible|
-     ansible.playbook = "playpen/ansible/vagrant-playbook.yml"
- end
-
+ # vagrant plugin install vagrant-cachier for some caching goodness
```

Comment:
Done

I just removed install instructions, since they belong in our docs (and they're already in our docs)